training_toolbox/__init__.py

```diff
@@ -0,0 +1,8 @@
+from os import path, environ
+import sys
+
+
+# import slim
```

Comment:
Is it debug related code?

Reply:
nope
its predefined imports instead PYTHONPATH usage

now you need not add paths to slim and research via PYTHONPATH, our an extrernal folder will be used by default

Reply:
It seems you understand me wrong :)
My question was about `# import slim`.